### PR TITLE
Code of Conduct Iteration #1

### LINKED ISSUE #3: about/more content
- [ ] server rules
- [ ] "about this server" / "code of conduct" copy
- [ ] Moderation policy

* who owns the data/content?
* minimum age?

## Research
From replies to [this toot](https://hachyderm.io/@mattwynne/109446934937833108).

- [Living room rules](https://notebook.drmaciver.com/posts/2020-07-14-13:42.html) | [Application](https://docs.google.com/document/d/1y3DkPGuAZiGbeNYlwje6jTlFxz9peIGnjSuZTwF4NGM/edit#heading=h.fi3d1ow65r1y)
- Chaos.social’s [rules](https://blog.chaos.social/rules) ([context](https://blog.chaos.social/2022/11/29/changes))
- https://berlincodeofconduct.org/
- https://infosec.exchange/about
- https://sfba.social/about | https://hub.sfba.social/code-of-conduct/
- https://ottawa.place/about/more 
- https://hachyderm.io/about/more
- https://friend.camp/about/more 
- https://wiki.social.coop/rules-and-bylaws/Code-of-conduct.html  
- https://wiki.social.coop/How-to-make-the-fediverse-your-own.html 
- https://github.com/stumpsyn/policies/blob/master/citizen_code_of_conduct.md

### PR DESCRIPTION
**Pull request by mattwynne**
Fixes #3

### CODE REVIEW

File: CodeOfConduct.md
Comment (tillkruss) [reply CodeOfConduct.md:13]: Is there anyone local who could advise us?

Comment (mattwynne) [reply CodeOfConduct.md:13]: Yeah. I lifted this from ottawa.place and I think we should say something, somewhere. But you're right it's a bit empty at the moment.

In the context of a Code of Conduct, I guess we could say something about, and I don't necessarily have the right words for this, that we are anti-colonialist by default and welcome feedback on how we can incorporate more indigenous culture into this document and the way this server is run.

I have a couple of people I could ask.

Comment (mattwynne) [reply CodeOfConduct.md]: I've taken it out for now. I've made https://github.com/nelson-social/community/issues/17 so we don't forget about it.

File: CodeOfConduct.md
Comment (blaine) [inline CodeOfConduct.md:45]: Maybe Expected -> Encouraged just to lean more on carrot than stick?

File: CodeOfConduct.md
Comment (blaine) [inline CodeOfConduct.md:71]: Would be good to reiterate "if you don't like it, you are very welcome to leave, link to instances.social" here.

File: CodeOfConduct.md
Comment (blaine) [inline CodeOfConduct.md:89]: I'd like to add a note that we're aware that we're three privileged white dudes (though only one born-canadian! Yay im/migration!) and while we are happy to continue to pay for the service and volunteer our labour, we would like to divest ourselves of power and would gladly welcome volunteers (and eventually paid staff, if appropriate) to help or lead decision making. (Or something roughly to that effect?)

**Review by blaine — APPROVED** [review]
GESET!

File: CodeOfConduct.md
Comment (mattwynne) [reply CodeOfConduct.md]: Nice idea! Done in 1bbf80906e2ad2e2bc12d7b84a6e71e87bdb574c

I wonder if we could also add something like that to the `About nelson.social` section

**Comment by mattwynne** [comment]
@tillkruss I think this is just waiting for your seal of approval now

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:9]: We use "encourage" twice here, can we use a stronger word?

Expect, demand, require?

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:15]: @mattwynne are you happy with this?

IMO we can use it as is for now, but it lacks a call to action/accountability.

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:21]: Love this.

Genuine question: Does it mean I can't joke about people who believe the earth is flat or that they are a mermaid with x-ray vision?

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:23]: The reference of "following ground rules", what does it refer to? The next sentence or "guiding principle"?

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:43]: Can we say:

"temporary suspension or permanent ban"?

Those would be the mastodon terms to avoid confusion and use DSL.

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:47]: "Notify a community organizer via direct message, or email admin@nelson.social"

Maybe that to make it less formal to always use email? It would be quicker for us as well to react to.

Comment (tillkruss) [reply ExtendedDescription.md:47]: Maybe we should also mention that people can "report" toots directly (on mobile and web app).

Comment (tillkruss) [reply ExtendedDescription.md:47]: Maybe there is a level 1 "please take a look at this, it offensive" and a level 2 "DM/email the team"?

Since mastodon has moderation/reporting built in we may as well use it.

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:83]: I just spend a week doing legal. We should say Canada or British Columbia, since that's where to court would take place.

**Review by tillkruss — CHANGES_REQUESTED** [review]
This is beautiful @matt. I've nitpicked a few minor things. If you like respond, or I can also leave commit-suggestions to make it easy to merge for you, if you agree with them.

**Comment by mattwynne** [comment]
Nice one.

If you feel confident in your suggestion, I suggest you just push changes to the branch and resolve the comment.

I'm offline now until Friday night and any loose ends I can tie up then.

File: ExtendedDescription.md
Comment (mattwynne) [reply ExtendedDescription.md:9]: Sure, well spotted. I think this was copy @blaine used ChatGPT to generate!

File: ExtendedDescription.md
Comment (mattwynne) [reply ExtendedDescription.md:15]: I think it's good enough for now. Can we continue the discussion in #17?

File: ExtendedDescription.md
Comment (mattwynne) [reply ExtendedDescription.md:21]: I think that would be absolutely fine. Which of the traits on that list would you be worried that would come under? Religion?! 😀

File: ExtendedDescription.md
Comment (mattwynne) [reply ExtendedDescription.md:23]: Yeah we should be more explicit and say "guidelines in this document" or some such. Good catch!

File: ExtendedDescription.md
Comment (mattwynne) [reply ExtendedDescription.md:47]: I thought about that but don't know mastodon well enough yet to be able to describe it. It would be great if you want to try and add some detail here.

File: ExtendedDescription.md
Comment (tillkruss) [reply ExtendedDescription.md:15]: Perfect.

File: ExtendedDescription.md
Comment (tillkruss) [reply ExtendedDescription.md:21]: I'm really just curious what radical inclusivity looks like in the real world. Maybe better to be discussed over a cuppa 🇬🇧

**Comment by mattwynne** [comment]
There's good stuff here on accessibility, moderation etc

https://hub.907.life/code-of-conduct/

File: ExtendedDescription.md
Comment (tillkruss) [reply ExtendedDescription.md:23]:
```suggestion
As a community, we expect any participant to act according to the guidelines in this document. They ensure that everyone within the community is clear about the behaviour that is expected of them, and that newcomers know what kind of environment they can expect to find here.
```

File: ExtendedDescription.md
Comment (tillkruss) [reply ExtendedDescription.md:43]:
```suggestion
If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary suspension or permanent ban from the community without warning.
```

File: ExtendedDescription.md
Comment (tillkruss) [reply ExtendedDescription.md:47]:
```suggestion
If you are subject to or witness unacceptable behavior, please use the "Report user" functionality of Mastodon. If the situation is beyond unacceptable behavior, please notify any of the [Team](#team) via direct message. We are here to help you. Your reports will be taken seriously, treated in confidence and will not be dismissed or argued with.
```

File: ExtendedDescription.md
Comment (tillkruss) [inline ExtendedDescription.md:126]: Let's use standard links that work anywhere.

```suggestion
* [@blaine](https://nelson.social/@blaine)
* [@matt](https://nelson.social/@matt)
* [@till](https://nelson.social/@till)
```

File: ExtendedDescription.md
Comment (tillkruss) [reply ExtendedDescription.md:83]:
```suggestion
* Anything in violation of our [Terms of Service](https://nelson.social/terms) including posting anything illegal in British Columbia or Canada.
```

**Comment by tillkruss** [comment]
> If you feel confident in your suggestion, I suggest you just push changes to the branch and resolve the comment.

I left commit suggestions for you to merge. If you're not familiar, you can commit them one-by-one or in a batch easily with the click of a button.

File: ExtendedDescription.md
Comment (tillkruss) [reply ExtendedDescription.md:9]:
```suggestion
We expect our members to be respectful and considerate of others, and to use this space to help build a stronger, more connected community in Nelson. We have a simple set of rules for the space that help the team that runs nelson.social ensure that we can be welcoming for everyone. Please read them, and reach out to us if you encounter any problems, or have any questions.
```

**Comment by mattwynne** [comment]
No proselethising

**Comment by mattwynne** [comment]
I'm merging this, and I'll put it on the site. I have a couple of other things I'd like to do, based on IRL conversations over the weekend, but I will do those as separate PR(s).

1) remove religion from the list of "personal traits" (beliefs are a choice, unlike sexuality, ethnicity etc.)
2) add a "No proselethising" rule
3) review https://hub.907.life/code-of-conduct/ for ideas

**Comment by mattwynne** [comment]
OK this is live at https://nelson.social/about/more#about-nelson-social

**Comment by tillkruss** [comment]
I'm in favor of removing the spirituality/religion from that list.

**Pull request merged**
